packages/accordion/README.md

```diff
@@ -67,4 +67,5 @@ Read the [contributing guide](https://vaadin.com/docs/latest/guide/contributing/
 
 Apache License 2.0
 
-Vaadin collects development time usage statistics to improve this product. For details and to opt-out, see https://github.com/vaadin/vaadin-usage-statistics.
+Vaadin collects development time usage statistics to improve this product.
```

Comment:
```suggestion
Vaadin collects usage statistics at development time to improve this product.
```

Reply:
Updated. Will do the same for all the `vaadin-` packages in a separate PR.